kinesis/src/main/scala/akka/stream/alpakka/kinesis/Errors.scala

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.kinesis
```

Comment:
It might make sense to put this into a different package (eg. `kinesisscheduler` or `kinesiskcl`) so that is is kept separate just as `kinesisfirehose`.